src/Jobs/IndexJob.php

```diff
@@ -47,15 +47,10 @@ public function __construct(
 
     public function setup(): void
     {
```

Comment:
do you think we could could use a fetcher here too? Might be nice to have one way of doing it?